test/typescript/prosemirror-tables.ts

```diff
@@ -21,3 +28,8 @@ const tableRect: TableRect = {
   map,
   table,
 };
+
+EditorState.create({
+  doc: table,
+  selection: CellSelection.create(table, 0),
+});
```

Comment:
I added it to the test, but currently test-typescript.js is not working correctly.

https://github.com/MH4GF/prosemirror-tables/blob/371af26f950612c7a4ca0046a94a65ce970c0eb5/test/test-typescript.js

Here's the PR we're going to fix: https://github.com/ProseMirror/prosemirror-tables/pull/177